src/Authenticator/LoginHandler.php

```diff
@@ -99,14 +106,23 @@ public function mfa()
     /**
      * Provides information about the current Member's MFA state
      *
+     * @param HTTPRequest $request
      * @return HTTPResponse
      */
-    public function getSchema()
+    public function getSchema(HTTPRequest $request)
     {
         try {
             $member = $this->getMember();
             $schema = SchemaGenerator::create()->getSchema($member);
-            return $this->jsonResponse($schema);
+            return $this->jsonResponse(
+                $schema + [
+                    'endpoints' => [
+                        'register' => $this->Link('mfa/register/{urlSegment}'),
+                        'login' => $this->Link('mfa/login/{urlSegment}'),
+                        'complete' => $this->Link('mfa/complete'),
```

Comment:
This seems like a concern of the SchemaGenerator - you could add `getAuthEndpoints(RequestHandler $handler)` or something?

Reply:
I'm not sure I agree. This is likely to be different for the registration screens in the member profile section? I could implement a way to set the endpoints for the generator and then ask the generator to give me the compiled schema but it seems kind of round-about to me?

Reply:
No problem. It just seems a bit odd to me that the schema generator isn't generating all of the scehma

Reply:
Yeah fair enough. Maybe there's something that satisfies both arguments. Perhaps raise a separate issue we can follow up on later?

Reply:
Nah